e2e/libs/host/harvester.py

```
@@ -111,3 +113,5 @@ def power_on_node(self, node_name):
                 logging(f"Getting vm status failed with error {e}")
             time.sleep(self.retry_interval)
         assert started, f"Expected vm {vm_id} to be started but it's not"
+
+        self.node.wait_for_node_up(power_on_node_name)
```

Ruff (0.8.0)

Undefined name `power_on_node_name`

(F821)